packages/odata-service-inquirer/src/prompts/connectionValidator.ts

```diff
@@ -387,11 +441,18 @@ export class ConnectionValidator {
                 isSystem,
                 odataVersion
             });
-
+            LoggerHelper.logger.debug(`ConnectionValidator.validateUrl() - status: ${status}; url: ${url}`);
+            // Since an exception was not thrown, this is a valid url
+            this.validity.urlFormat = true;
+            this._validatedUrl = url;
```

Comment:
Do we need to ensure that even if a http 200 is returned, it's not a saml HTML response.

Reply:
I think that should be dealt with by a lower level module, i.e. axios extension?